pkg/controller/androidvariant/configmap.go

```diff
@@ -0,0 +1,26 @@
+package androidvariant
+
+import (
+	pushv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/push/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *ReconcileAndroidVariant) createAndroidConfigMap(androidVariant *pushv1alpha1.AndroidVariant) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "android-variant-configmap",
+			Namespace: "mobile-developer-console",
+		},
+		Data: createConfigMapData(androidVariant),
```

Comment:
```suggestion
		Data: map[string]string{
			"senderId":      androidVariant.Spec.SenderId,
			"variantId":     androidVariant.Status.VariantId,
			"variantSecret": androidVariant.Spec.ServerKey,
		}
```

^this removes the need for the `createConfigMapData` function.